src/Elmish.WPF/Binding.fs

```diff
@@ -391,13 +412,19 @@ type Binding private () =
   /// <summary>Creates a two-way binding.</summary>
   /// <param name="get">Gets the value from the model.</param>
   /// <param name="set">Returns the message to dispatch.</param>
+  /// <param name="wrapDispatch">
+  ///   Wraps of the dispatch function with additional behavior,
```

Comment:
Typo, change `Wraps of the` to `Wraps the`, here and other similar places.

While you're at it, I usually aim for a line length of 80-90 characters (though I don't care that much if you'd rather keep it this way).

I skimmed the rest of the file; if you know you did anything unique further below, please let me know (everything looked equal).

Reply:
Fixed the documentation typo (via find replace) in a force pushed commit.

I added this documentation as a suggestion.  I don't mind replacing it with anything that you would like instead.

Yes, everything else was straightforward.  For example, I fixed all compile errors in the tests by passing in `id`.

Reply:
I am indifferent to merging with these separate commits or squashing first.  Whatever you would prefer.

Reply:
> Yes, everything else was straightforward. For example, I fixed all compile errors in the tests by passing in `id`.

Oh, I realized later that maybe I could have provided more insight here.  One of the compile errors in the tests was because the compiler was unable to determine which method to resolve.

Does that change your mind about the new version being 3.3.0 instead of 4.0.0?

Even if you are (attempting to) follow [SemVer](https://semver.org/), I think that it is vague enough that there is ample room for disagreement about whether or not an upgrade such as this, one that can introduce a compiler error, must increment the major version.  I am unsure if SemVer considers this to be a "backwards incompatible change".  I think merely incrementing the minor version (as you did) is fine.

Reply:
Thanks for bringing this to my attention. I think that in this particular it's not a problem; it's just that the `fail` test helper is completely generic, and thus it can't distinguish between a call using no `canExecute` and only `wrapDispatch`, and a call using `canExecute` but not `wrapDispatch`. In real-world usage, I expect `canExecute` to be sufficiently typed that this won't cause a problem.

For the record: Overload resolution problems are IMHO the main argument against using overloads. The ergonomics are good enough that I think it's a good tradeoff, though. I'm inclined to consider any introduction of overload resolution errors a breaking change, but they can be hard to catch unless you're prescient and know exactly what to look for. Having tests for all overloads and parameter combinations certainly help.